interface_specifications/pe_openapi/apis/v1/c1_pe_definition_get.md

```diff
@@ -0,0 +1,35 @@
+### Presentation Definition GET
+
+`GET /pe/v1/{thid}/definition/{definition_id}`
+
+
+#### Parameters
+
+Path Variables:
+
+* `thid`: Thread ID associated with the presentation exchange. e.g. `f1ca8245-ab2d-4d9c-8d7d-94bf310314ef`
+* `definition_id`: definition_id returned from the previous http request `POST /pe/v1/definition/`. e.g. `32f54163-7166-48f1-93d8-ff217bdb0653`
+
+
+#### Response
+
+* `200`: Success
+
+  Response Body
+```json
+    {
+      "presentation_definition": {
```

Comment:
thid in the body (see earlier discussion)?

Reply:
fixed